src/Bubble.vala

```diff
@@ -174,15 +174,16 @@ public class Notifications.Bubble : AbstractBubble {
             attach (body_label, 1, 1);
 
             if (notification.actions.length > 0) {
-                var action_area = new Gtk.ButtonBox (Gtk.Orientation.HORIZONTAL) {
-                    layout_style = Gtk.ButtonBoxStyle.END
-                };
+                var action_area = new Gtk.Box (Gtk.Orientation.HORIZONTAL, 0);
```

Comment:
One property that buttonbox has by default that we don't get here for free is `homogeneous`:
```suggestion
                var action_area = new Gtk.Box (Gtk.Orientation.HORIZONTAL, 0) {
                    homogeneous = true
                };
```

Reply:
oh yeah right.. lemme commit this. Thanks!